frontend/src/components/NewRuleForm.vue

```diff
@@ -66,9 +66,10 @@ const handleGenerationRulesChanged = (rules: GenerationRule[]) => {
         <h4>
           <EditableName
             name="name"
-            value="New Rule 1"
-            button-class=""
-            input-class="form-control-lg"
+            value=""
+            button-class="fs-3"
```

Comment:
The `fs-3` class here makes the icon a bit lower than the text. Removing the class makes it look properly aligned. How about removing it?